test/integration/smoke/test_network_ipv6.py

```diff
@@ -665,14 +209,16 @@ def setUp(self):
         self.services = self.testClient.getParsedTestDataConfig()
         self.apiclient = self.testClient.getApiClient()
         self.dbclient = self.testClient.getDbConnection()
+        self.thread = None
         self.cleanup = []
         return
 
     def tearDown(self):
         try:
+            if self.thread and self.thread.is_alive():
+                self.thread.join(5*60)
```

Comment:
with this extra code we run the test of the cleanup not being reached. Please see the construct I sugested in other tests to apply it here as well.